### PR TITLE
change master to main

### PR DESCRIPTION
**Pull request by zhaojizhuang**
change branch from `master` to `main` for link in docs

### CODE REVIEW

**Review by vaikas — APPROVED** [review]
Thanks for doing this!
/lgtm

File: docs/eventing/sources/_index.md [hunk 1]
```diff
@@ -53,28 +53,28 @@ kn source list
...
 [DockerHubSource](https://github.com/tom24d/eventing-dockerhub) | v1alpha1 | None | Retrieves events from [Docker Hub Webhooks](https://docs.docker.com/docker-hub/webhooks/) and transforms them into CloudEvents for consumption in Knative.
 [FTP / SFTP](https://github.com/vaikas-google/ftp) | Proof of concept | None | Watches for files being uploaded into a FTP/SFTP and generates events for those.
-[Heartbeat](https://github.com/Harwayne/auto-container-source/tree/master/heartbeat-source) | Proof of Concept | None | Uses an in-memory timer to produce events as the specified interval. Uses AutoContainerSource for underlying infrastructure.
+[Heartbeat](https://github.com/Harwayne/auto-container-source/tree/main/heartbeat-source) | Proof of Concept | None | Uses an in-memory timer to produce events as the specified interval. Uses AutoContainerSource for underlying infrastructure.
```
Comment (markusthoemmes) [inline docs/eventing/sources/_index.md:72]: This results in a 404

/hold

File: docs/eventing/sources/_index.md [hunk 2]
```diff
@@ -53,28 +53,28 @@ kn source list
...
 [Konnek](https://konnek.github.io/docs/#/) | Active Development | None | Retrieves events from cloud platforms (like AWS and GCP) and transforms them into CloudEvents for consumption in Knative.
-[K8s](https://github.com/Harwayne/auto-container-source/tree/master/k8s-event-source) | Proof of Concept | None | Brings Kubernetes cluster events into Knative. Uses AutoContainerSource for underlying infrastructure.
-[RedisSource](https://github.com/lionelvillard/eventing-redis/tree/master/source) | v1alpha1 | None | Brings Redis Stream into Knative.
+[K8s](https://github.com/Harwayne/auto-container-source/tree/main/k8s-event-source) | Proof of Concept | None | Brings Kubernetes cluster events into Knative. Uses AutoContainerSource for underlying infrastructure.
```
Comment (markusthoemmes) [inline docs/eventing/sources/_index.md:74]: 404 here too :(

**Comment by markusthoemmes** [comment]
Two stray 404s, other than that LGTM. Thanks!

File: docs/eventing/sources/_index.md (hunk 2 above)
Comment (zhaojizhuang) [reply docs/eventing/sources/_index.md]: done

File: docs/eventing/sources/_index.md (hunk 1 above)
Comment (zhaojizhuang) [reply docs/eventing/sources/_index.md]: done

**Comment by zhaojizhuang** [comment]
done! @markusthoemmes Thanks

> Two stray 404s, other than that LGTM. Thanks!

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

File: docs/eventing/sources/_index.md [hunk 3]
```diff
@@ -53,28 +53,28 @@ kn source list
...
+[Amazon Cognito User](https://github.com/triggermesh/aws-event-sources/tree/main/cmd/awscognitouserpoolsource/README.md) | Supported | TriggerMesh | Registers for events from [Amazon Cognito](https://aws.amazon.com/cognito/) user pools.
+[Amazon DynamoDB](https://github.com/triggermesh/aws-event-sources/blob/main/cmd/awsdynamodbsource/README.md) | Supported | TriggerMesh | Reads records from an [Amazon DynamoDB](https://aws.amazon.com/dynamodb/) stream.
+[Amazon Kinesis](https://github.com/triggermesh/aws-event-sources/tree/main/cmd/awskinesissource/README.md) | Supported | TriggerMesh | Reads records from an [Amazon Kinesis](https://aws.amazon.com/kinesis/) stream.
+[Amazon SNS](https://github.com/triggermesh/aws-event-sources/tree/main/cmd/awssnssource/README.md) | Supported | TriggerMesh | Subscribes to messages from an [Amazon SNS](https://aws.amazon.com/sns/) topic.
+[Amazon SQS](https://github.com/triggermesh/aws-event-sources/tree/main/cmd/awssqssource/README.md) | Supported | TriggerMesh | Consumes messages from an [Amazon SQS](https://aws.amazon.com/sqs/) queue.
```
Comment (evankanderson) [inline docs/eventing/sources/_index.md:64]: These all seem to have been renamed back to `master`.

**Review by evankanderson — COMMENTED** [review]
One more case where the `master` -> `main` transition seems to have been changed/rolled back, though it looks like there are GitHub redirects in place.

/approve
/hold

(`/hold cancel` if you're fine putting in as-is, or change the triggermesh links and I'll re-`/lgtm`)

**Comment by zhaojizhuang** [comment]
@evankanderson I have put `main` back to `master` for triggermesh repository

> One more case where the `master` -> `main` transition seems to have been changed/rolled back, though it looks like there are GitHub redirects in place.
> 
> /approve
> /hold
> 
> (`/hold cancel` if you're fine putting in as-is, or change the triggermesh links and I'll re-`/lgtm`)

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**